packages/dds/tree/src/feature-libraries/editable-tree/README.md

```diff
@@ -0,0 +1,12 @@
+# Editable Tree
```

Comment:
I recently normalized all these sub-remes to just match the directory/package name with their title exactly.
```suggestion
# editable-tree
```